astrogrism/config/HST/create_HST_reffiles.py

```diff
@@ -0,0 +1,19 @@
+from pathlib import Path
+
+from astrogrism.config.HST.reference_file_generators._generate_specwcs import create_grism_specwcs  # noqa
+from astrogrism.config.HST.reference_file_generators._generate_wavelengthrange import create_tsgrism_wavelengthrange  # noqa
+# from astrogrism.config.HST.reference_file_generators.reference_file_generators._generate_distortion import create_distortion  # noqa
+
+
+def create_reference_files(conffile, hst_grism, outpath=Path.cwd(), outbasename=None):
+    if not outbasename:
+        outbasename = Path(conffile).name
+    create_tsgrism_wavelengthrange(outname=str(Path(outpath) / (
```

Comment:
I think that's something I developed from my CS 201 class about optimizations, that being avoiding the need to create a variable to hold the value, rather than just creating it and passing it immediately. But the resulting code IS satisfyingly shorter and cleaner, I can't pass that up 😅

Reply:
Having never taken a CS class I'm going on pure aesthetics here. Maybe I need to update my taste to match best practices!

Reply:
Haha I never said I was *TAUGHT* to do this, rather I picked it up when they said "variables take memory, be mindful about your variables" and I found out that by constructing arguments live, I didn't need to make a variable! Who knows if it actually does anything!?